mysql/test_utils.go

```diff
@@ -12,17 +12,17 @@ func GetMySQLTestClient(c *opc.Config) (*MySQLClient, error) {
 	// Build up config with default values if omitted
 
 	if c.IdentityDomain == nil {
-		domain := os.Getenv("OPC_IDENTITY_DOMAIN")
+		domain := os.Getenv("ORACLEPAAS_IDENTITY_DOMAIN")
```

Comment:
We can keep these as `OPC_...` since these values are the same between OPC and ORACLEPAAS environments. I would like to update these with a more generic name but we're locked in since Terraform uses the `OPC_...` naming conventions.

Reply:
Ok. I've reverted all these back to OPC_...